orga/tests/acceptance/missions-list_test.js

```diff
@@ -0,0 +1,45 @@
+import { module, test } from 'qunit';
+import { currentURL } from '@ember/test-helpers';
+import { visit } from '@1024pix/ember-testing-library';
+import { setupApplicationTest } from 'ember-qunit';
+import authenticateSession from '../helpers/authenticate-session';
+import { createUserWithMembershipAndTermsOfServiceAccepted, createPrescriberByUser } from '../helpers/test-init';
+
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+import setupIntl from '../helpers/setup-intl';
+
+module('Acceptance | Missions List', function (hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+  setupIntl(hooks);
+
+  module('When prescriber is not logged in', function () {
+    test('it should not be accessible by an unauthenticated prescriber', async function (assert) {
+      // when
+      await visit('/missions');
+
+      // then
+      assert.deepEqual(currentURL(), '/connexion');
+    });
+  });
+
+  module('When prescriber is logged in and has mission management feature', function () {
+    test('it should be accessible to the list of the missions', async function (assert) {
```

Comment:
"user should access to the list of the missions" ou "the list of the missions should be accessible"